### PR TITLE
docs/variants/novacustom_nv4x/post_install.md: add Nvidia drivers section

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki <kamil.pokornicki@3mdeb.com>

### CODE REVIEW

File: docs/variants/novacustom_nv4x/post_install.md [hunk 1]
```diff
@@ -26,26 +26,43 @@ enable the touchpad hotkey to work under Linux, follow the steps below:
 After executing these steps, it should be possible to enable and disable the
 touchpad using the touchpad hotkey (Fn+F1) on the keyboard when using GNOME.
 
+## Nvidia drivers (Ubuntu 22.04)
+
+For proper functioning of the sleep mode on Ubuntu 22.04, it is required to
```
Comment (sulewskiprzemyslaw) [inline docs/variants/novacustom_nv4x/post_install.md:31]: Maybe the word `working` should be better than `functioning`?

File: docs/variants/novacustom_nv4x/post_install.md [hunk 2]
```diff
@@ -26,26 +26,43 @@ enable the touchpad hotkey to work under Linux, follow the steps below:
...
+install additional Nvidia drivers.
+
+1. Install drivers by executing the following command on Ubuntu 22.04:
```
Comment (sulewskiprzemyslaw) [inline docs/variants/novacustom_nv4x/post_install.md:34]: Maybe the better option is to write, where the command should be executed (in the terminal), than on which system *that info we have in the header).

File: docs/variants/novacustom_nv4x/post_install.md [hunk 3]
```diff
@@ -26,26 +26,43 @@ enable the touchpad hotkey to work under Linux, follow the steps below:
...
+
+    ```bash
+    sudo apt install nvidia-driver-515 nvidia-dkms-515
+    ```
+
+1. Reboot the device to apply changes by executing the following command:
```
Comment (sulewskiprzemyslaw) [inline docs/variants/novacustom_nv4x/post_install.md:40]: As above

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/novacustom_nv4x/post_install.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/variants/novacustom_nv4x/post_install.md]: replaced

File: docs/variants/novacustom_nv4x/post_install.md (hunk 3 above)
Comment (Pokisiekk) [reply docs/variants/novacustom_nv4x/post_install.md]: replaced

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**